app/src/main/java/org/mifos/mobilebanking/presenters/RegistrationVerificationPresenter.java

```diff
@@ -69,7 +70,7 @@ public void onCompleted() {
                     @Override
                     public void onError(Throwable e) {
                         getMvpView().hideProgress();
-                        getMvpView().showError(MFErrorParser.errorMessage(e));
+                        getMvpView().showError(context.getString(R.string.fail_verify));
```

Comment:
Do not show a custom error msg by using context.getString(R.string.fail_register), as MFErrorParser.errorMessage(e) contains important error msg which should be displayed to the user so revert back to it.